{"name": "困人处置列表", "status": "passed", "attachments": [{"name": "log", "source": "9c0b9f16-d45f-4518-aa9c-0e215abbc913-attachment.txt", "type": "text/plain"}], "start": 1692256587761, "stop": 1692256587895, "uuid": "0b0cb7d2-56f9-4cb0-8dbc-d2139e933079", "historyId": "e9533ca1bce9dcd222ffc25dce3a836e", "testCaseId": "1307c5a2fbf84e6aa7a4e7b10201eae2", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "19796-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}